src/containers/Explorer/index.js

```diff
@@ -1,15 +1,40 @@
 import React from 'react';
 import FilterNav from './FilterNav';
 import ExplorerBody from './ExplorerBody';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { actions } from 'public-modules/Bounties';
+import { toggleFromParam } from 'utils/locationHelpers';
 import styles from './Explorer.module.scss';
 
-const Explorer = props => {
-  return (
-    <div className={`${styles.explorerContainer}`}>
-      <FilterNav />
-      <ExplorerBody />
-    </div>
-  );
-};
+class Explorer extends React.Component {
+  constructor(props) {
+    super(props);
+    const { resetState, history, location, load, toggleStageFilter } = props;
+    resetState();
+    toggleStageFilter('active');
+    load(true);
+  }
 
-export default Explorer;
+  render() {
+    return (
+      <div className={`${styles.explorerContainer}`}>
```

Comment:
I don't believe this needs the `${}` and could be passed in just as `{styles.explorerContainer}`. Mentioning this here because when I read the string formatting I automatically expect there to be hard coded mentions in the string too. Thoughts?

Reply:
Yeah - probably a remnant of when I was passing multiple classes at some pt.